tests/python/tank_test/mock_appstore.py

```diff
@@ -289,9 +293,10 @@ def _find_latest(self):
             self._type, self.get_system_name()
         )
         latest = "v0.0.0"
-        for version in versions:
-            if LooseVersion(version) > LooseVersion(latest):
-                latest = version
+        with suppress_known_deprecation():
```

Comment:
Wow, I love this function! 
Can you add a comment to say for which Python version we had this?

Reply:
Agree with that , that context manager is great